src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,26 @@
+@mixin center-positioning {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 30px;
+  height: 30px;
+
+  transform: translate(-50%, -50%);
+}
+
+@mixin arrow-styling {
+  content: "";
+
+  position: absolute;
+  height: 20px;
+  top: 5px;
+  left: 15px;
+
+  border-radius: 8px;
+
+  transform-origin: left;
+
+  animation-name: circle;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
```

Comment:
It seems to me that it would be better to write everything in one line, but this is not an error i think
`animation: circle infinite linear`